src/components/kpis-bar/kpis-bar.less

```diff
@@ -83,52 +64,99 @@
       }
       @media screen and (max-width: 675px) {
         top: -35px;
-        right: 20px;
+        right: 67px;
         bottom: 0;
       }
     }
   }
 
-  .title {
-    .text-center {
-      line-height: 3;
-      font-variant: small-caps;
-      font-size: larger;
+
+  .content {
+    padding: 15px;
+    min-height: @kpi-max-height + 30px;
+    // overflow: hidden;
+    background-color: @kpis-bar-background-color;
+    box-shadow: @kpis-bar-box-shadow;
+
+    &.empty { min-height: 50px; }
+
+    transition: all .2s ease-in;
+    @media screen and (min-width: 725px) {
+      padding: 0 60px;
     }
-    .badge {
-      position: absolute;
-      top: 8px;
-      left: -10px;
-      background-color: #d1e55c;
-      color: #4f5959;
-      box-shadow: 0px 1px 3px -1px;
+    @media screen and (max-width: 675px) {
+      margin-top: 18px;
     }
-    .no-kpi-templates {
-      color: #afadb5;
-      font-weight: 500;
+    .impac-kpi {
+      .column-gutter(5px);
+
+      @media screen and (min-width: 1200px) {
+        padding-left: 20px;
+        padding-right: 20px;
+      }
+
+      .kpi.add {
+        .kpi-show .caption {
+          white-space: inherit;
+        }
+      }
     }
-  }
 
-  .add-bar {
-    border-bottom: dashed 1px;
-  }
+    &.hide-animate {
+      min-height: 0;
```

Comment:
This seems to have no effect, I think you're looking for `max-height`, but even that will not reduce the height of the kpi-bar and all it's children to 0.